apps/total-typescript/src/pages/team/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import Layout from 'components/app/layout'
+import {find, get, isNull, isString} from 'lodash'
+import {convertToSerializeForNextResponse} from '@skillrecordings/commerce-server'
+import {GetServerSideProps} from 'next'
+import {getPurchasedProduct} from 'team/get-purchased-product'
+import BuyMoreSeats from 'team/buy-more-seats'
+import {UserGroupIcon, TicketIcon} from '@heroicons/react/outline'
+import {useSession} from 'next-auth/react'
+import {getCurrentAbility} from 'ability/ability'
+import {getToken} from 'next-auth/jwt'
+import {getSdk} from '@skillrecordings/database'
+import Card from 'team/card'
+import InviteTeam from 'team'
+
+export const getServerSideProps: GetServerSideProps = async ({req}) => {
+  const token = await getToken({req})
+  const ability = getCurrentAbility({user: token as any})
```

Comment:
the call signature of `getCurrentAbility` changed and "token is user" and it's now a "bag of options"

The primary reason for pushing `ability` package into local space is because TT users don't === purchasers and we have different content types with different rules.

local `ability` will need to be reconciled with `packages/ability`